### ISSUE TITLE
Not working with Ionic 1.3.13

### ISSUE THREAD

**Issue by kevbaker** [israelidanny/ion-google-place#36]
I am trying to get ion-google-place working with Ionic 1.3.13.

After following the README instructions I am not seeing the location popup.

I am also not getting any errors. 

Any idea on how to debug this?

**Comment by softBarbarian** [comment]
Did you include css to the head in index.html?

**Comment by kevbaker** [comment]
Ok now I am getting the overlay which looks great, but I am unable to click on any of the items as I type. It is dynamically updating but not allowing me to tap on an item to close the modal.

**Comment by kevbaker** [comment]
Digging into the `ion-google-place` code a bit more it seems like the `selectLocation` method on the popup scope is not being called.

I do have my <selectLocation> tag in a modal. I'll see if that is somehow related... perhaps a z-index issue?

**Comment by kevbaker** [comment]
Yes that seems to be the problem. When I launch the ion-google-place popup from a non-ion-modal window it works perfectly.

Any thoughts on how to address this?

**Comment by kevbaker** [comment]
Ah found the fix in another [issue thread #4 ](https://github.com/israelidanny/ion-google-place/issues/4)
Is there a reason this fix, adding the `modal` class to the `ion-google-place-container`, hasn't been merged?

**Comment by kevbaker** [comment]
Create a quick [pull request](https://github.com/israelidanny/ion-google-place/pull/37) for this if you'd like to merge it.

**Comment by exoer** [comment]
I struggled with getting the data out from the directive in the controller. My solution was to use a object in the ng-model

     <ion-google-place placeholder="Search City, Country" ng-model="search_data.location" > </ion-google-place>

in the controller

    $scope.search_data = {
          location: ''
        };

    $scope.$watch('search_data.location', function(newVal, oldVal) {
            console.log('watch', newVal, oldVal)
            if (newVal.hasOwnProperty('formatted_address')) {
                $scope.search_city();
            }
        });

**Comment by sbasi5576** [comment]
this worked for me. I used $scope.apply to assign the output to a variable.
